src/main/scala/cromwell/engine/package.scala

```diff
@@ -49,7 +49,9 @@ package object engine {
    * created
    */
   case class WorkflowDescriptor(id: WorkflowId, sourceFiles: WorkflowSourceFiles) {
-    private val conf = ConfigFactory.load
+    // TODO: Extract this from here (there is no need to reload the configuration for each workflow)
```

Comment:
Yes, we should actually have a global `Configuration` object that loads it once and for all, and every other class takes it form here to extract whatever sub-conf they need.
This could be a ticket I think.